src/chainparams.cpp

```diff
@@ -106,6 +106,17 @@ void CChainParams::UpdateBudgetParameters(int nMasternodePaymentsStartBlock, int
     consensus.nSuperblockStartBlock = nSuperblockStartBlock;
 }
 
+void CChainParams::UpdateSubsidyAndDiffParams(int nMinimumDifficultyBlocks, int nHighSubsidyBlocks, int nHighSubsidyFactor)
+{
+    consensus.nMinimumDifficultyBlocks = nMinimumDifficultyBlocks;
+    consensus.nHighSubsidyBlocks = nHighSubsidyBlocks;
+    consensus.nHighSubsidyFactor = nHighSubsidyFactor;
+}
+
+void CChainParams::UpdateLLMQChainLocks(Consensus::LLMQType llmqType) {
+    consensus.llmqChainLocks = llmqType;
```

Comment:
Unrelated to this PR, but an observation: the variable name `llmqChainLocks` is a little confusing. Maybe should be `llmqTypeChainLocks` or similar.